CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -559,7 +559,7 @@ local function CreateSettingsHub()
 			--to update, or we can just avoid using it so soon.
 			RunService.Heartbeat:wait()
 
-			if shouldShowBottomBar() then
+			if this.Visible and shouldShowBottomBar() then
```

Comment:
I thought about it but I don't think it makes sense. You probably shouldn't even be asking if you can show the bottom bar if the menu isn't already visible.